src/main/kotlin/no/nav/medlemskap/regler/v1/ReglerForAndreStatsborgere.kt

```diff
@@ -12,8 +12,8 @@ class ReglerForAndreStatsborgere(
         ytelse: Ytelse
 ) : Regler(ytelse) {
 
-    override fun hentRegelflyt(): Regelflyt {
-        return konklusjonUavklart(ytelse)
+    override fun hentRegelflyter(): List<Regelflyt> {
+        return listOf(konklusjonUavklart(ytelse))
```

Comment:
Det er vanskelig å henge helt med i svingene her, hvorfor er dette nå en liste med regelflyter?

Reply:
Skjønner at det er vanskelig å henge med på det. Tenkte å presentere refaktoreringsløpet for dere i morgen. ReglerForNorskeStatsborgere er et eksempel der man trenger å kjøre flere regelflyter for å kjøre alle relevante regler. Man ønsker å kjøre stillingsprosent-regelen selv om den foregående regelflyten gir en nei-konklusjon.